apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -204,10 +205,23 @@ export class PnpmOptionsConfiguration {
    */
   public readonly resolutionStrategy: ResolutionStrategy;
 
+  /**
+   * If true, then rush will block "rush install" when the lockfile has been manually edited.
+   *
+   * @remarks
+   * This causes "rush install" to fail when it is determined that the the lockfile was manually edited. This
+   * behavior can be ignored by providing the "--bypassPolicy" argument to "rush install". This is done to
+   * ensure that the lockfile is free from human error.
+   *
+   * The default value is false.
```

Comment:
Once you've finalized these docs, you need to update [assets/rush-init/rush.json](https://github.com/microsoft/rushstack/blob/63dfae043f32cb2065c68e38b636afe6cc32d6a2/apps/rush-lib/assets/rush-init/rush.json#L1) which is the primary way that we document **rush.json** settings.